src/components/site/Search.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+
+const Search = props => (
+  <form className="dg_search-container" id="search-bar">
```

Comment:
can we cahnge this id to "search-form" please?

Reply:
also please add a label for the input based on the w3 recommendation https://www.w3.org/WAI/tutorials/forms/labels/#hiding-the-label-element.

Reply:
done